src/generators.cpp

```diff
@@ -301,7 +307,25 @@ void Generator::ComputeLogits() {
   search_->ApplyRepetitionPenalty(search.repetition_penalty);
 }
 
+void Generator::SetRuntimeOptionsConfig(const char* key, const char* value) {
+  // TODO: Need a better way to handle different keys
+  // We can create a config manager to host all configurations and do comparison at that point
+  if (strcmp(key, "terminate_session") == 0) {
```

Comment:
These strings ("terminate_session") need to be stored as constants and documented in a separate file in this repo and shipped with the genai pkg so that users can discover these options.

Reply:
Thanks, I have added the details and usage in documents/Runtime_option.md file.

Reply:
I was thinking of a file like [this](https://github.com/microsoft/onnxruntime/blob/c5b6be045ff58b706390bd7504e7d865e451a689/include/onnxruntime/core/session/onnxruntime_session_options_config_keys.h) so that users can include it while calling the APIs and use the constants therein. This also allows the constants to be used elsewhere in the code base. Let's fix it later.

Reply:
Ok, I have created a task for this.